src/ProtocolGateway.Core/Instrumentation/SafePerformanceCounter.cs

```diff
@@ -4,16 +4,28 @@
 namespace Microsoft.Azure.Devices.ProtocolGateway.Instrumentation
 {
     using System;
+#if !NETSTANDARD1_3
     using System.Diagnostics;
+#endif
 
-    public sealed class SafePerformanceCounter
+    public sealed class SafePerformanceCounter : IPerformanceCounter
```

Comment:
As we discussed it is ok to change the public API, I removed SafePerformanceCounter from netstandard